### PR TITLE
ci: use era-compiler-llvm-test tag in integration tests if exists

### PR DESCRIPTION
**Pull request by antonbaliasnikov**
# Code Review Checklist

## Purpose

Use `era-compiler-llvm-test` tag in `era-compiler-tester` repository for candidate benchmarks and integration tests if this tag exists. If it's not, default to the default branch (`main`).

### CODE REVIEW

File: .github/workflows/benchmarks-integration-tests.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    inputs:
+      llvm_build_type:
+        description: "LLVM build type: debug | release"
+        required: true
+        default: "release"
+      compiler_tester_reference_branch:
+        description: "compiler-tester branch to use as a benchmark reference"
+        required: true
+        default: "main"
+      compiler_tester_candidate_branch:
```
Comment (hedgar2017) [inline .github/workflows/benchmarks-integration-tests.yml]: It is a tag, not branch, isn't it?
Let's rephrase it to: defaults to branch `main` if the tag doesn't exist.

**Review by hedgar2017 — CHANGES_REQUESTED** [review]

File: .github/workflows/benchmarks-integration-tests.yml (hunk 1 above)
Comment (antonbaliasnikov) [reply .github/workflows/benchmarks-integration-tests.yml]: Fixed, please, take a look

**Review by hedgar2017 — APPROVED** [review]

**Review by hedgar2017 — CHANGES_REQUESTED** [review]
Shall we not skip the CI here to check that the new scripts work?

**Review by hedgar2017 — APPROVED** [review]
